java/integTest/src/test/java/glide/cluster/ClusterTransactionTests.java

```diff
@@ -68,15 +66,10 @@ public void WATCH_transaction_failure_returns_null() {
     @SneakyThrows
     public void info_simple_route_test() {
         ClusterTransaction transaction = new ClusterTransaction().info().info();
-        ClusterValue<Object>[] result =
-                clusterClient.exec(transaction, RANDOM).get(10, TimeUnit.SECONDS);
+        Object[] result = clusterClient.exec(transaction, RANDOM).get(10, TimeUnit.SECONDS);
```

Comment:
10, TimeUnit.SECONDS do you need this I thought we were removing it.

Reply:
Thanks, removed